nipype/pipeline/engine/tests/test_newnode_neuro.py

```diff
@@ -0,0 +1,120 @@
+from .. import NewNode, NewWorkflow
+from ..auxiliary import Function_Interface
+
+#dj niworkflows vs ...??
+from niworkflows.nipype.interfaces.utility import Rename
+import niworkflows.nipype.interfaces.freesurfer as fs
+
+from ...interfaces.freesurfer import PatchedConcatenateLTA as ConcatenateLTA
+
+Name = "example"
+DEFAULT_MEMORY_MIN_GB = None
+Inputs = {}
+
+# wf = Workflow(name, mem_gb_node=DEFAULT_MEMORY_MIN_GB,
+#               inputs=['source_file', 't1_preproc', 'subject_id',
+#                       'subjects_dir', 't1_2_fsnative_forward_transform',
+#                       'mem_gb', 'output_spaces', 'medial_surface_nan'],
+#               outputs='surfaces')
+#
+#dj: why do I need outputs?
+wf = NewWorkflow(name=Name, mem_gb_node=DEFAULT_MEMORY_MIN_GB, inputs=Inputs)
```

Comment:
inputs/outputs set the appropriate fields of the `Workflow` compute object. such that workflows themselves are seen as interfaces.

Reply:
didn't think about it